puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_virt_who_configure.yaml

```diff
@@ -1,5 +1,28 @@
 - project:
     name: foreman_virt_who_configure
     defaults: plugin
+    dbs: 'postgresql'
+    parameters:
+      - string:
+          name: branch
+          default: master
+          description: "Branch to run test suite on"
+      - pr_parameters
+      - string:
+          name: foreman_branch
+          default: develop
+          description: "Foreman branch to run test suite against"
+      - string:
+          name: foreman_pr_git_url
+          default:
+          description: 'Foreman it URL to PR to test, e.g. git://github.com/user/foo.git'
+      - string:
+          name: foreman_pr_git_ref
+          default:
+          description: 'Foreman Git ref for PR to test, usually the branch name'
     jobs:
       - 'test_plugin_{name}_{branch}'
+    builders:
+      - clone_foreman
+      - test_katello
+      - test_foreman_virt_who_configure
```

Comment:
Does foreman_virt_who link in to Katello enough to make running test_katello worth while?